### PR TITLE
Add support for Nicaragua provider - v1

### PR DESCRIPTION
**Pull request by xamanu**
This is a PR to the `v1` branch, as this is the currently official release on Google Play Store.

### CODE REVIEW

**Comment by xamanu** [comment]
There is still either the inclusion in navitia.io pending, or the link to the community's navitia is missing. Please wait until this is confirmed, before merging in. However review and feedback is highly appreciated.

**Comment by xamanu** [comment]
Hereby I explicitly state that I accept the [Contributor License Agreement](https://github.com/grote/Transportr/blob/master/CLA.md) whenever contributing to Transportr.

**Comment by grote** [comment]
Hey, I am traveling now and can only merge/review this in a bit more than a week. This PR seems to be missing the PTE upgrade to your staging fork. Did you try to compile and run this? It shouldn't compile without the PTE upgrade.

File: build.gradle [hunk 1]
```diff
@@ -75,7 +75,7 @@ dependencyVerification {
 			'com.google.code.findbugs:jsr305:bec0b24dcb23f9670172724826584802b80ae6cbdaba03bdebdef9327b962f6a',
 			'com.squareup.okio:okio:734269c3ebc5090e3b23566db558f421f0b4027277c79ad5d176b8ec168bb850',
 			'com.google.errorprone:error_prone_annotations:cb4cfad870bf563a07199f3ebea5763f0dec440fcda0b318640b1feaa788656b',
-			'com.google.j2objc:j2objc-annotations:40ceb7157feb263949e0f503fe5f71689333a621021aa20ce0d0acee3badaa0f',
+			//'com.google.j2objc:j2objc-annotations:40ceb7157feb263949e0f503fe5f71689333a621021aa20ce0d0acee3badaa0f',
```
Comment (xamanu) [inline build.gradle]: Is this related to this topic?

Comment (ialokim) [reply build.gradle]: It didn't compile on my machine, not really sure why. But I think we could revert this change.

Comment (grote) [reply build.gradle]: I am not yet sure what is going on here. For now, please just keep this line removed.

**Comment by grote** [comment]
Thanks  again for this! It seems that this is still using a very old PTE version and shouldn't even compile like this. @ialokim or @xamanu does any of you want to switch v1 to your PTE fork? What's the deadline for getting 1.1.9 released?

@cla-bot[bot] check

**Comment by ialokim** [comment]
> It seems that this is still using a very old PTE version

That's true, in order to not change a lot of the core code, I've added Nicaragua following the same workaround you already used for Spain, New Zealand and Brazil.

> and shouldn't even compile like this

It does compile indeed, I have the working example running on my personal phone.

> @ialokim or @xamanu does any of you want to switch v1 to your PTE fork?

I could give it a try, but probably not until next week. I think it would be easier to publish 1.1.9 like this (if it compiles for you, obviously) instead of "wasting" the time changing core functions of the v1 branch which will never be used again. Not sure, what do you think?

**Comment by xamanu** [comment]
Given to the principle to stick to smallest PRs possible and the important time constraint, I prefer to follow @ialokim's approach to keep it as it is (using the same workaround as Spain, New Zealand and Brazil). In version 2 the newest PTE is supported then.

**Comment by xamanu** [comment]
Given to the principle to stick to smallest PRs possible and the important time constraint, I prefer to follow @ialokim's approach to keep it as it is (using the same workaround as Spain, New Zealand and Brazil). In version 2 the newest PTE is supported then.

**Comment by grote** [comment]
Ahh I was looking for the PTE upgrade and didn't notice you added the provider file outside of PTE. That works since this branch is deprecated anyway.

Still, I fixed the CI and also tried to fix the resulting merge conflicts, but it seems GitHub is still not allowing me to merge this. Would you mind rebasing this on latest `v1` branch and maybe squash the commits into one while you are at it?

**Comment by grote** [comment]
Ok turned out, I could merge this with squashing, but just not rebasing.

Thanks again!

**Comment by ialokim** [comment]
Great to have it working! Thanks a lot! 🎉
